climpred/metrics.py

```diff
@@ -155,17 +122,138 @@ def get_metric_function(metric):
                 'msss'
                 'nmae'
                 'uacc'
+                'msss_murphy'
+                'bias'
+                'bias_slope'
+                'conditional_bias'
+                'std_ratio'
+                'crps'
+                'crpss'
+                'less'
                 """)
         return metric
 
 
-# TODO: Do we need wrappers or should we rather create wrappers for skill score
-#       as used in a specific paper: def Seferian2018(ds, control):
-#       return PM_compute(ds, control, metric=_ppp, comparison=_m2e)
-def _ppp(ds, control, comparison, running=None, reference_period=None):
+# wrap xskillscore metrics to work with comparison argument
+def _pearson_r(forecast, reference, dim='svd', comparison=None):
+    return pearson_r(forecast, reference, dim=dim)
+
+
+def _pearson_r_p_value(forecast, reference, dim='svd', comparison=None):
```

Comment:
To allow for comparison argument to be passed

Reply:
also now `pearson_r_p_value` is just another `metric`. simplifies the code a lot.